### DISCUSSION TITLE
why i see this error

### DISCUSSION THREAD

**Discussion by Ali3447** [community/community#121585] [Programming Help]
### Select Topic Area

Question

### Body

invalid object key - customData (argument="transaction:customData", value={"data":"0x2cc4081e00000000000000000000000000000000000000000000000000000000000000600000000000000000000000000000000000000000000000000000000000f08eea000000000000000000000000000000000000000000000000000000006630787a00000000000000000000000000000000000000000000000000000000000000010000000000000000000000000000000000000000000000000000000000000020000000000000000000000000000000000000000000000000000000000000006000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000011c37937e0800000000000000000000000000000000000000000000000000000000000000000010000000000000000000000000000000000000000000000000000000000000020000000000000000000000000814a23b053fd0f102aeeda0459215c2444799c7000000000000000000000000000000000000000000000000000000000000000800000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000010000000000000000000000000000000000000000000000000000000000000000600000000000000000000000005300000000000000000000000000000000000004000000000000000000000000b0477a50cafe4ce6d3f8f64a81cde4f42963755a00000000000000000000000000000000000000000000000000000000000000020000000000000000000000000000000000000000000000000000000000000000","to":"0x80e38291e06339d10AAB483C65695D004dBD5C69","gasLimit":{"type":"BigNumber","hex":"0x2dc6c0"},"from":"0xB0477A50cAfE4cE6d3F8f64A81CDE4f42963755A","value":{"type":"BigNumber","hex":"0x11c37937e08000"},"customData":{"gasPerPubdata":50000,"paymasterParams":{"paymasterInput":"0x949431dc000000000000000000000000000000000000000000000000000000000000800a00000000000000000000000000000000000000000000006c6b935b8bbd40000000000000000000000000000000000000000000000000000000000000000000600000000000000000000000000000000000000000000000000000000000000020000000000000000000000000000000000000000000000000000000000000000f"}}}, code=INVALID_ARGUMENT, version=properties/5.7.0)

**Comment by LiteBrite82** [comment]
Thanks for posting in the GitHub Community, @Ali3447 !

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Feed category is solely related to conversations around the GitHub product "Feed". This question should be in the Programming Help category. I've gone ahead and moved it for you. Good luck!

**Comment by vinihsr** [comment]
This error message indicates an "invalid object key" related to "customData" in a transaction argument. It suggests that there's an issue with the provided custom data format or structure.

**Comment by Aadil-H-W** [comment]
what vinihsr said is true. The error message you provided seems to indicate that there's an issue with an object key named customData. It appears that you're trying to pass an object with this key, but there might be some problem with the format or structure of the data.

to fix this you could try:
1.Error Handling: Look for any error handling mechanisms provided by the API or library. The error message you provided includes a code=INVALID_ARGUMENT, which suggests that the data you're passing might be invalid according to the expected arguments.

2. Data Format: Ensure that the data you're passing for the customData key is in the correct format and follows any requirements or constraints specified by the API or library you're using.

3.Object Structure: Double-check the structure of the object you're passing. Make sure it's a valid JSON object and that all keys and values are correctly formatted.

**Comment by smilefox-lab** [comment]
I think it is encountering an error related to an "invalid object key - customData" .
